.github/workflows/python_test.yml

```diff
@@ -0,0 +1,35 @@
+name: Python package
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11"]
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install ruff pytest
+          if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Lint with ruff
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
+          ruff --format=github --select=E9,F63,F7,F82 --target-version=py37 .
+          # default set of ruff rules with GitHub Annotations
+          ruff --format=github --target-version=py37 .
+      - name: Test with pytest
+        run: |
+          pytest
+      - name: Run test.py
+        run: |
+          python test.py
```

Comment:
we should move the `test.py` file into `tests/test_reminders.py`

Reply:
I did this, but I have an error while importing reminder.py

Reply:
I'm looking for a way to solve this problem

Reply:
I wrote this to import `reminder.py` in `test_reminders.py` :
```py
from .. import reminder
rem = reminder.Reminder() 
```

Reply:
I did this :

```py
import os
import sys

parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), ".."))
sys.path.append(parent_dir)

from reminder import Reminder
````

And it's worked, however the pre-commit blocked it 

```
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

tests/test_reminders.py:10:1: E402 module level import not at top of file
```